### ISSUE TITLE
paper runtimes when shift-clicking

### ISSUE THREAD

**Issue by ShizCalev** [tgstation/tgstation#51861]
caused by #51819 

this is because examine respects a message to be returned for a chat message. 
```

[02:40:41] Runtime in mob.dm, line 433: Cannot execute null.Join().
verb name: Examine (/mob/verb/examinate)
usr: ShizCalev/(monkey (718))
usr.loc: (Atmospherics (127,97,2))
src: the monkey (718) (/mob/living/carbon/monkey)
src.loc: the floor (127,97,2) (/turf/open/floor/plasteel)
call stack:
the monkey (718) (/mob/living/carbon/monkey): Examine(the pamphlet (/obj/item/paper/pamphlet))
the pamphlet (/obj/item/paper/pamphlet): ShiftClick(the monkey (718) (/mob/living/carbon/monkey))
the monkey (718) (/mob/living/carbon/monkey): ShiftClickOn(the pamphlet (/obj/item/paper/pamphlet))
the monkey (718) (/mob/living/carbon/monkey): ClickOn(the pamphlet (/obj/item/paper/pamphlet), "icon-x=16;icon-y=12;left=1;shi...")
the pamphlet (/obj/item/paper/pamphlet): Click(the floor (128,97,2) (/turf/open/floor/plasteel), "mapwindow.map", "icon-x=16;icon-y=12;left=1;shi...")
ShizCalev (/client): Click(the pamphlet (/obj/item/paper/pamphlet), the floor (128,97,2) (/turf/open/floor/plasteel), "mapwindow.map", "icon-x=16;icon-y=12;left=1;shi...")
```

**Comment by ShizCalev** [comment]
@WarlockD

**Comment by ShizCalev** [comment]
Fixed by #52147
